src/Form/index.tsx

```diff
@@ -688,19 +693,31 @@ function Form({
             clientInstance.submitCustom(castValues, false);
           }
 
-          // User is authenticating. auth hook will set the initial stepKey once auth has finished
-          if (authState.redirectAfterLogin) return;
+          const onboardingExists = areThereOnboardingQuestions(
+            session.integrations
+          );
+          const isFormComplete = session.form_completed;
+
+          // can I drop onboardingExists?
```

Comment:
pretty sure I don't need to check if onboardingExists, because all that really matters is if the form is complete. Forms without onboarding will be marked complete during login process, forms with onboarding will be marked complete after finishing the questions